src/main/java/picard/arrays/ArraysCallingMetricAccumulator.java

```diff
@@ -76,14 +77,15 @@
                         detail.P95_GREEN = p95Green;
                         detail.P95_RED = p95Red;
                         detail.SCANNER_NAME = scannerName;
+                        detail.PIPELINE_VERSION = pipelineVersion;
                         return detail;
                     }, true);
 
     ArraysCallingMetricAccumulator(DbSnpBitSetUtil.DbSnpBitSets dbsnp) {
         this.dbsnp = dbsnp;
     }
 
-    public void setup(final VCFHeader vcfHeader) {
+    public void setup(final VCFHeader vcfHeader, final String pipelineVersion) {
```

Comment:
Given the way this class is used, I don't see why the code in `setup()` isn't done in the constructor directly.

Reply:
The code has been rewritten to not pass pipelineVersion to CollectArraysVariantCallingMetrics, so this suggestion isn't germane to the PR (and probably should be done elsewhere).